app/controllers/placements/schools/school_contacts/edit_school_contact_controller.rb

```diff
@@ -0,0 +1,54 @@
+class Placements::Schools::SchoolContacts::EditSchoolContactController < Placements::ApplicationController
+  before_action :set_school
+  before_action :set_school_contact
+  before_action :set_wizard
+
+  helper_method :step_path, :current_step_path, :back_link_path, :add_mentor_path
+
+  def new
+    @wizard.reset_state
+    @wizard.setup_state
+    redirect_to step_path(@wizard.first_step)
+  end
+
+  def edit; end
+
+  def update
+    if !@wizard.save_step
+      render "edit"
+    else
+      @wizard.update_school_contact
+      Placements::SchoolSlackNotifier.school_onboarded_notification(@school).deliver_later
+      @wizard.reset_state
+      redirect_to placements_school_path(@school), flash: { success: t(".success") }
+    end
+  end
+
+  private
+
+  def set_school
+    school_id = params.require(:school_id)
+    @school = current_user.schools.find(school_id)
+  end
+
+  def set_school_contact
+    @school_contact = @school.school_contact
+  end
```

Comment:
This controller assumes that the school has a contact. Do you think we should raise a `404 Not Found` error if it doesn't? 🤔

```suggestion
  def set_school_contact
    @school_contact = @school.school_contact
    
    raise ActiveRecord::RecordNotFound if @school_contact.nil?
  end
```

Otherwise I'm assuming something further downstream will explode and result in a `500 Internal Server Error`. I wonder if this is a useful case to catch at this point instead.

What do you think?